src/win/pipe.c

```diff
@@ -747,9 +766,14 @@ int uv_pipe_bind2(uv_pipe_t* handle,
     req->next_pending = NULL;
   }
 
-  err = uv__convert_utf8_to_utf16(name, &handle->name);
-  if (err)
-    return err;
+  /* TODO(bnoordhuis) Add converters that take a |length| parameter. */
```

Comment:
Remove these comments now?

Reply:
They're still relevant because they'll let us avoid the malloc().